content/04.results.md

```diff
@@ -41,3 +41,19 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Discriminative Model Calibration
+
+![
+Calibration plots for the discriminative model.
```

Comment:
Similar to my comment on the other figure. Walk me through the interpretation first (I think that here it's that temperature scaling doesn't usually improve calibration, except maybe for DaG - at least that's what I'm getting as the main message from your paragraph below).